src/views/preview/comment/comment.scss

```diff
@@ -150,7 +150,7 @@
                 content: "";
             }
 
-            &.comment-bubble-deleted {
+            &.comment-bubble-deleted, .comment-bubble-reported {
```

Comment:
this is a bit confusing to me. Am I right that this means:

an element with both .comment-bubble AND .comment-bubble-deleted
OR
.comment-bubble-reported (that appears under an element with) .comment-bubble
?

Reply:
Eek yeah that is wrong, I'll fix it

Reply:
Fixed in 997d0b5